modules/appeals_api/config/flipper/enabled_features.yml

```diff
@@ -0,0 +1,21 @@
+common:
+  - decision_review_hlr_email
+  - decision_review_nod_email
+  - decision_review_sc_email
+  - decision_review_icn_updater_enabled
```

Comment:
@cilestin Can you confirm whether having the `decision_review_icn_updater_enabled` enabled in all environments makes sense? It is currently only enabled in dev.

Reply:
It should probably only be enabled in dev. We've disconnected the job that was using it from running for the time being, but we may want to reuse it later.

Reply:
Thank you! This update is complete.